source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -7,24 +7,24 @@ title: Variables and Aliases
 # {% fa fa-graduation-cap %} What You'll Learn
 
 - How to deal with async commands
-- What Aliases are, how they simplify your code
+- What Aliases are and how they simplify your code
 - Why you rarely need to use variables with Cypress
 - How to use Aliases for objects, elements and routes
 {% endnote %}
 
 # Return Values
 
-New users to Cypress may initially find it challenging to work with the asynchronous nature of our API's.
+New users to Cypress may initially find it challenging to work with the asynchronous nature of our APIs.
```

Comment:
Corrected a handful of such in this file: APIs is the plural of API.

Some style-guides advocate for using an apostrophe with acronyms so the style with apostrophes isn't wrong. I changed it to APIs in this file _only_ to be consistent with the _Introduction To Cypress_ which uses APIs. However, there are other files in the documentation that use API's with the apostrophe: I can add 'em to this PR to get all of the documentation consistent.

> Totes cool if you also think that the style using the apostrophe is what you want.